### PR TITLE
[setup] move setup configuration to setup.cfg

### LINKED ISSUE #151: Drop MANIFEST.in
Use [setup.py/data_files](https://docs.python.org/3/distutils/setupscript.html#installing-additional-files) instead

### PR DESCRIPTION
**Pull request by apalala**
fixes #151, #193
original pull request in  #174 (by @KOLANICH https://github.com/KOLANICH)

### CODE REVIEW

**Comment by apalala** [comment]
I will probably remove the automatic generation of `tastu/_version.py`.  TatSu releases have been almost a year apart, and there are many branches and pull requests open in between.

I prefer to set the version manually upon a release, using the guidelines from [Semantic Versioning](https://semver.org).

File: setup.cfg [hunk 1]
```diff
@@ -1,8 +1,47 @@
...
-universal = 1
-
 [metadata]
+name = TatSu
+version = 5.5.0
+author = Juancarlo Añez
+author_email = apalala@gmail.com
+maintainer = Juancarlo Añez
+maintainer_email = apalala@gmail.com
+license = BSD License
```
Comment (KOLANICH) [inline setup.cfg:8]: BTW, should be a SPDX id.

File: setup.cfg [hunk 2]
```diff
@@ -1,8 +1,47 @@
...
 license_file = LICENSE.txt
```
Comment (KOLANICH) [inline setup.cfg:9]: Detected fully automatically, not needed.

File: tatsu/_version.py [hunk 3]
```diff
@@ -1 +1 @@
-from tatsu._config import __version__  # noqa
+__version__ = '5.7.0'
```
Comment (KOLANICH) [inline tatsu/_version.py:1]: This file should be `.gitignore`d

Comment (apalala) [reply tatsu/_version.py:1]: What happens if there is no version tag on the branch?

Comment (KOLANICH) [reply tatsu/_version.py:1]: Error.

Comment (apalala) [reply tatsu/_version.py:1]: I don't understand how day-to-day work can happen with that. Tags are usually applied upon a release.

Comment (KOLANICH) [reply tatsu/_version.py:1]: Easily. Shallow clones with insufficient depth.

Comment (apalala) [reply tatsu/_version.py:1]: @KOLANICH, would you please post a new PR with the changes you suggest? :-)

**Pull request merged**